docs/SOP.md

```diff
@@ -11,6 +11,8 @@ This document contains standard operating procedures (SOPs) for the OBO Foundry
 
 ## SOPs
 - [New Ontology Requests](#NOR)
+- [Reviewing Ontologies for OBO Membership](#ROOM)
```

Comment:
anchors automatically get generated based on the titles, so you could do
```suggestion
- [Reviewing Ontologies for OBO Membership](#reviewing-ontologies-for-obo-membership)
```

Reply:
Thanks for tip!

Reply:
I like fixed anchors better, just so we don't have to think about what happens when we change a title..